### PR TITLE
BZ is done.

### PR DESCRIPTION
**Pull request by BZhawaii**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,7 +38,10 @@ var add = function (x, y) {
 }
 ```
 
--- YOUR ANSWER HERE --
+ES5.
+NO.
```
Comment (mesprague) [inline README.md]: This is anonymous. The function is assigned to a variable called add, but has no name itself

File: README.md [hunk 2]
```diff
@@ -70,8 +80,9 @@ var obj = {
   }
 }
 ```
-
--- YOUR ANSWER HERE --
+ES6
```
Comment (mesprague) [inline README.md]: ES5

File: README.md [hunk 3]
```diff
@@ -97,5 +110,10 @@ for ( console.log('Bob'); (console.log('Curly') === undefined); console('Mo') )
 console.log(i)
 ```
 
--- YOUR ANSWER HERE --
+Bob
+Curly
+1 
+Mo
+1
+
```
Comment (mesprague) [inline README.md:120]: Bob
Curly
1
Mo
Curly
2
2

Please experiment with for loops like Carlos did in class.

File: README.md [hunk 4]
```diff
@@ -97,5 +110,12 @@ for ( console.log('Bob'); (console.log('Curly') === undefined); console('Mo') )
 console.log(i)
```
Comment (mesprague) [inline README.md:110]: Thanks for giving it another run through!! Did you actually run this code? It still has the typo

**Comment by BZhawaii** [comment]
Hi Matt, I didn't actually run the code.  I thought we were supposed to do it in our heads.
I did run it just now, and got the same results that I wrote down.
BobCurly1MoCurly22

 

    On Tuesday, January 17, 2017 12:28 AM, Matthew Sprague <notifications@github.com> wrote:
 

 @mesprague commented on this pull request.In README.md:> @@ -97,5 +110,12 @@ for ( console.log('Bob'); (console.log('Curly') === undefined); console('Mo') )
 console.log(i)
Thanks for giving it another run through!! Did you actually run this code? It still has the typo—

**Pull request closed without merging**